ci-operator/step-registry/upi/vsphere/proxy/https/upi-vsphere-proxy-https-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: upi-vsphere-proxy-https
+  steps:
+  - ref: ipi-install-rbac
+  - ref: ipi-conf
+  - ref: upi-conf-vsphere-proxy-https
```

Comment:
Can we reuse `ipi-conf-vsphere-proxy` here?

Reply:
`ipi-conf-vsphere-proxy` is used to configure cluster with http proxy. QE has another CI profile to test cluster with https proxy, so I add new ref `upi-conf-vsphere-proxy-https` here.

Reply:
I moved `upi-conf-vsphere-proxy-https` to `ipi-conf-vsphere-proxy-https`, this may be more clearly for future usage if needed.
http_proxy configuration: `ipi-conf-vsphere-proxy`
https_proxy configuration: `ipi-conf-vsphere-proxy-https`